crafter-search-solr-provider/solr/configsets/crafter_configs/conf/managed-schema

```diff
@@ -175,6 +175,7 @@
     <dynamicField name="*_d"  type="pdouble" indexed="true"  stored="true"/>
     <dynamicField name="*_ds" type="pdoubles" indexed="true"  stored="true"/>
     <dynamicField name="*_dmv" type="pdoubles" indexed="true"  stored="true"/>
+    <dynamicField name="*_raw" type="string" indexed="true"  stored="true"/>
```

Comment:
this shouldn't be a string, this should be text. The body still needs to be analyzed/tokenized.

Reply:
why? the original content should only be used when it is needed to display in a client, any search done in there will be likely wrong because it will include all HTML tags & attributes. All queries should use the `_html` that has only meaningful text

Reply:
Discussed it with Russ and we couldn't think of a use case where you would search the field, you're right.